_33_Search_in_Rotated_Sorted_Array.java

```diff
@@ -0,0 +1,148 @@
+package com.java.binaryserach.leetcode;
+
+import java.util.Scanner;
```

Comment:
Try to submit the problem on leetcode and check if the solution is generalized

Reply:
I did, the package is mentioned for my eclipse workspace i did submit it without this line. Thank you for looking into this :)